src/security/engines/BrowserCryptoEngine.js

```diff
@@ -2,29 +2,25 @@ import KeyStoreCryptoEngine from './KeyStoreCryptoEngine';
 import BrowserKeyStore from './BrowserKeyStore';
 
 /**
- * BrowserCryptoEngine: Implements the CryptoEngine interface.
+ * Crypto engine for the browser, it handles signatures, verifications,
+ * and keys are stored in IndexedDB using the following schema
  *
- * Crypto engine to handle signatures, verifications, and key storage, on browsers. Uses
- * LocalStorage as the storage location, and handles storage for multiple members at once.
- * Uses the following schema:
- *
- * schemaVersion: 0.2,
- *
- * activeMemberId: "m:12345:678", // member ID
- * members: {
- *  "m:12345:678": { // member ID
- *    LOW: {
+ * KEY: memberId (e.g. 'm:12345:678')
+ * VALUE (JS object):
+ * {
+ *      LOW: {
  *          id: 456, // key ID
- *          algorithm: ED25519,
+ *          algorithm: ECDSA,
  *          level: LOW,
- *          publicKey: "jlfgjlrsjfglgfja", // Crypto.strKey(public key)
- *          secretKey: "ljkfsfjkfgjlfjjf", // Crypto.strKey(secret key)
+ *          publicKey: "jlfgjlrsjfglgfja", // Crypto.strKey(publicKey)
```

Comment:
nit: change the "jlfgjlrsjfglgfja" it looks unprofessional

Reply:
lol